lib/dependenciesInstall.js

```diff
@@ -56,15 +57,15 @@ module.exports = function(context) {
             loaded: false
         }, function(err) {
             if (err) {
-                console.log('cordova-plugin-livereload: An error occurred while loading npm');
-                reject('cordova-plugin-livereload: npm load failed.');
+                logger.show('An error occurred while loading npm');
```

Comment:
Handled with this commit: https://github.com/omefire/cordova-plugin-livereload/commit/c7c9ae64907ead75f9860bf4112bf7f867119346